gadget.Dockerfile

```diff
@@ -36,7 +36,7 @@ FROM docker.io/kinvolk/traceloop:202109280218336f4efc as traceloop
 # https://github.com/kinvolk/bcc/commit/1d78d60b2319aa36e6e8c120966d9502ba0a0b75
 # See BCC section in docs/CONTRIBUTING.md for further details.
```

Comment:
Please update also the comment.

BTW, as I said in the `--columns` PR, let's consider for a future PR to change this comment to avoid having to update it each time and remove the possibility to miss it.

Reply:
> Please update also the comment.

Yeah, will do after https://github.com/kinvolk/bcc/pull/8 is merged. (I'll have to change both of them anyway, so I'll do it only after that pr is merged). 

> BTW, as I said in the `--columns` PR, let's consider for a future PR to change this comment to avoid having to update it each time and remove the possibility to miss it.

:+1: